client-src/elements/chromedash-gate-column.js

```diff
@@ -153,6 +162,21 @@ class ChromedashGateColumn extends LitElement {
     }
   }
 
+  handleSelectChanged() {
+    this.needsSave = true;
+  }
+
+  handleSave() {
+    // TODO(jrobbins): This should specify gate ID rather than gate type.
+    window.csClient.setApproval(
+      this.feature.id, this.gate.gate_type,
+      this.voteSelectRef.value.value)
```

Comment:
this.voteSelectRef.value.value or this.voteSelectRef.value?

Reply:
It is supposed to be `value.value`, even though it looks odd.  The first `value` is getting the value of the ref, which gives the textarea element.  The second `value` gets the value of that element, which is the text that the user typed.